File: app/src/main/java/in/hridayan/ashell/ui/CustomSearchView.java
package in.hridayan.ashell.ui;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.EditText;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import com.google.android.material.R;
import com.google.android.material.search.SearchView;

public class CustomSearchView extends SearchView {

  public CustomSearchView(@NonNull Context context) {
    super(context);
  }

  public CustomSearchView(@NonNull Context context, @Nullable AttributeSet attrs) {
    super(context, attrs);
  }

  public CustomSearchView(
      @NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
    super(context, attrs, defStyleAttr);
  }

  public EditText getSearchEditText() {
    return findViewById(R.id.open_search_view_edit_text);
  }
}

File: app/src/main/java/in/hridayan/ashell/ui/CustomSearchView.kt
package `in`.hridayan.ashell.ui

import android.content.Context
import android.util.AttributeSet
import android.widget.EditText
import com.google.android.material.R
import com.google.android.material.search.SearchView

class CustomSearchView : SearchView {
    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    )

    val searchEditText: EditText
        get() = findViewById<EditText>(R.id.open_search_view_edit_text)
}
